index.js

```diff
@@ -93,6 +94,10 @@ function buildResponseMessage(prDetails) {
     }
     if (!isValidPullRequestBody(prDetails.body)) {
         message += '1. PR Description\n';
+        if (process.env.ENABLE_KEYWORD_CHECKER != undefined
+            && process.env.ENABLE_KEYWORD_CHECKER.toLowerCase() === "true") {
```

Comment:
Oh I see, that'd definitely make things easier for me but might be a challenge for users to edit parameters though, since it'll all be on a single line. @damithc any thoughts on this?